target/classes/application.yml

```diff
@@ -0,0 +1,16 @@
+spring:
```

Comment:
don't include target folder.  ignore it on update.  Use your ide to tag the directory as ignored or not included in commit.